src/selection.h

```diff
@@ -1,383 +1,374 @@
-
-class Selection {

```

Comment:
why is this in the diff?

Reply:
I didn't see this diff in VSCode until I pushed the commit and saw it here.
I'm currently trying to understand what git thinks actually changed. Changing line endings to CRLF or LF in VSCode didn't fix it, the diff is still there.
Will do a hard push fixing this once I understand the problem.

Reply:
Looks like it was git just not liking the endings despite the changes not being there even when looking with hex editor.